build.gradle

```diff
@@ -12,3 +12,9 @@ allprojects {
         jcenter()
     }
 }
+
+apply plugin: 'checkstyle'
+
+checkstyle {
+    configFile = new File("${project.projectDir}", "checkstyle.xml")
+}
```

Comment:
- the `configFile` should be configured using the `rootDir` project variable
- you should explicitly set the checkstyle version used by gradle via `toolVersion` (note that some Gradle versions are incompatible with same checkstyle versions. e.g. Gradle v2.5 isn't playing nice with checkstyle v6.8)

````groovy
checkstyle {
    configFile = new File(rootDir, 'checkstyle.xml')
    toolVersion = '6.7'
}
````